packages/aila/src/lib/agentic-system/agents/sectionAgents/sectionAgentRegistry.ts

```diff
@@ -18,8 +19,13 @@ import { titleAgent } from "./titleAgent";
 
 export const createSectionAgentRegistry = ({
   openai,
+  customAgentHandlers,
 }: {
   openai: OpenAI;
+  customAgentHandlers: {
+    "starterQuiz--maths": SectionAgent<QuizV2>["handler"];
+    "exitQuiz--maths": SectionAgent<QuizV2>["handler"];
+  };
```

Comment:
Are these only custom because they use `this._chat.fullQuizService`? I wonder if we should just instantiate that inside the agentic system